### PR TITLE
add react-native version output for -v/--version

### PR DESCRIPTION
**Pull request by sunnylqm**
@satya164  @brentvatne

### CODE REVIEW

File: react-native-cli/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-cli",
-  "version": "0.1.7",
+  "version": "0.1.8",
```
Comment (satya164) [inline react-native-cli/package.json]: Can you remove the version change here?

File: react-native-cli/index.js [hunk 2]
```diff
@@ -221,8 +221,15 @@ function checkNodeVersion() {
...
   if (process.argv.indexOf('-v') >= 0 || process.argv.indexOf('--version') >= 0) {
-    var pjson = require('./package.json');
-    console.log(pjson.version);
+    var cliJson = require('./package.json');
+    console.log('react-native-cli: ' + cliJson.version);
+    try {
+      var rnJson = require(REACT_NATIVE_PACKAGE_JSON_PATH());
+      console.log('react-native: ' + rnJson.version);
+    }
+    catch (e) {
```
Comment (satya164) [inline react-native-cli/index.js]: Move catch to previous line, e.g.-  `} catch (e) {`

File: react-native-cli/index.js [hunk 3]
```diff
@@ -221,8 +221,15 @@ function checkNodeVersion() {
 
 function checkForVersionArgument() {
   if (process.argv.indexOf('-v') >= 0 || process.argv.indexOf('--version') >= 0) {
-    var pjson = require('./package.json');
-    console.log(pjson.version);
+    var cliJson = require('./package.json');
+    console.log('react-native-cli: ' + cliJson.version);
```
Comment (satya164) [inline react-native-cli/index.js]: Extra variable seems unnecessary, you can just do `console.log('react-native-cli: ' + require('./package.json').version)`

File: react-native-cli/index.js [hunk 4]
```diff
@@ -221,8 +221,15 @@ function checkNodeVersion() {
...
+    try {
+      var rnJson = require(REACT_NATIVE_PACKAGE_JSON_PATH());
+      console.log('react-native: ' + rnJson.version);
```
Comment (satya164) [inline react-native-cli/index.js]: console.log('react-native: ' + require(REACT_NATIVE_PACKAGE_JSON_PATH()).version)

File: react-native-cli/index.js [hunk 5]
```diff
@@ -221,8 +221,15 @@ function checkNodeVersion() {
...
+    }
+    catch (e) {
+      console.log('react-native: n/a - not currently inside a React Native project directory')
```
Comment (satya164) [inline react-native-cli/index.js]: `not currently inside a React Native project directory` -> `not inside a React Native project directory`

**Comment by satya164** [comment]
Thanks for the PR :)

**Comment by sunnylqm** [comment]
I've learned how to squash just now :)

**Comment by satya164** [comment]
@sunnylqm Awesome. thanks.

**Comment by sunnylqm** [comment]
Then how do you publish it to npm if version not changed? @satya164

**Comment by satya164** [comment]
@sunnylqm We'll change the version while publishing. We don't publish for every change.

**Pull request merged**